docs/pages/ktextbox.vue

```diff
@@ -2,15 +2,55 @@
 
   <DocsPageTemplate apiDocs>
     <DocsPageSection title="Overview" anchor="#overview">
-      For design guidance, see the page on <DocsInternalLink href="/textfields" text="text fields" />.
+      For detailed design guidance, refer to the page on <DocsInternalLink href="/textfields" text="text fields" />.
+    </DocsPageSection>
+
+    <DocsPageSection title="Input Types and Features">
+      <h3>Input with Label</h3>
+      <KTextbox label="Input with Label" />
+      <p>This text box includes a visible label, providing clear guidance and context to the user about the expected input.</p>
+
+      <h3>Input without Label</h3>
+      <KTextbox />
+      <p>This text box does not include a visible label. It's useful when the context of the input is obvious without a label, or if space constraints necessitate omitting it.</p>
+
+      <h3>Valid and Invalid Input</h3>
+      <KTextbox 
+        v-model="numericInput" 
+        label="Numbers Only" 
+        :invalid="!isNumeric(numericInput)" 
+        invalidText="Please enter a valid number." 
+      />
+      <p>This text box only accepts numeric input. If any non-numeric character is entered, it will be considered invalid.</p>
+
+      <h3>Disabled Input</h3>
+      <KTextbox label="Disabled Input" disabled />
+      <p>This text box is disabled and cannot be edited.</p>
```

Comment:
Here it might be good to also note that it cannot be focused by way of keyboard navigation, so perhaps:

"This text box is disabled. It cannot be edited or focused, so it will be skipped during keyboard navigation."

Thoughts @radinamatic @MisRob ?

Reply:
Makes sense to me